controllers/sysbench/job.go

```diff
@@ -32,11 +32,11 @@ func NewJob(cr *perfv1alpha1.Sysbench) *batchv1.Job {
 		Namespace: cr.Namespace,
 	}
 
-	sysbenchCmdLineArgs := []string{}
-	sysbenchCmdLineArgs = append(sysbenchCmdLineArgs, qsplit.ToStrings([]byte(cr.Spec.Options))...)
-	sysbenchCmdLineArgs = append(sysbenchCmdLineArgs, cr.Spec.TestName, cr.Spec.Command)
+	sysbenchArgs := []string{}
+	sysbenchArgs = append(sysbenchArgs, qsplit.ToStrings([]byte(cr.Spec.Args))...)
```

Comment:
same as above, sysbench prefix not required, appending to an empty one is suboptimal.